integreat_compass/cms/templates/offers/offer_form.html

```diff
@@ -73,7 +73,7 @@ <h2>{% translate "Offer details" %}</h2>
                                 <div>
                                     {{ choice.tag }}
                                     <label for="{{ choice.id_for_label }}"
-                                           class="px-4 py-2 text-lg bg-white rounded-full text-center shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
+                                           class="px-4 py-2 text-lg bg-white rounded-full text-center hover:shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
```

Comment:
```suggestion
                                           class="px-4 py-2 text-lg bg-tertiary font-normal rounded-full text-center hover:shadow-lg peer-checked:bg-primary peer-checked:text-gray-700 peer-checked:border-none">
```

Sorry, that I didn't think of this before. But without the shadow it's difficult to see the chips/tags now. Therefore I would suggest to adapt the look of the chips from "all offers"

Reply:
Maybe a custom class is already warranted? I think it's the same design in 3 places now, right?